src/ApprenticeAan/SFA.DAS.ApprenticeAan.Api.UnitTests/Controllers/LocationsControllerTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using SFA.DAS.ApprenticeAan.Api.Controllers;
+using SFA.DAS.ApprenticeAan.Application.Locations.Queries.GetAddresses;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.ApprenticeAan.Api.UnitTests.Controllers;
+
+public class LocationsControllerTests
+{
+    [Test]
+    [MoqAutoData]
+    public async Task GetAddresses_ReturnsAddresses(
+        GetAddressesQueryResult response,
+        [Frozen] Mock<IMediator> mockMediator,
+        [Greedy] LocationsController sut)
+    {
+        mockMediator
+            .Setup(m => m.Send(It.IsAny<GetAddressesQuery>(), It.IsAny<CancellationToken>()))
+            .ReturnsAsync(response);
+
+        var result = await sut.GetAddresses("thisIsAQuery");
+
+        result.As<OkObjectResult>().Should().NotBeNull();
+
+        result.As<OkObjectResult>().Value.Should().BeEquivalentTo(response);
```

Comment:
`mediatorMock.Verify(m => m.Send(It.IsAny<GetAdminMemberQuery>(), It.IsAny<CancellationToken>()));` can go here as seen [here](https://github.com/SkillsFundingAgency/das-aan-hub-api/blob/b5edde3a8e2081e9738a6cad615e9869da408d03/src/SFA.DAS.AANHub.Api.UnitTests/Controllers/AdminsControllerTests.cs#L101)

Reply:
I will add a separate test for this.